templates/fine-tune-llm_v2/README.md

```diff
@@ -85,4 +78,56 @@ You can edit the values, such as `context_length`, `num_epoch`, `train_batch_siz
 In addition, the deepspeed configs are provided in case you would
 like to customize them.
 
+### What's the full list of supported models?
+
+The following models can currently be fine-tuned with this template.
+
+- mistralai/Mistral-7B-Instruct-v0.1
+- mistralai/Mixtral-8x7b
+- meta-llama/Llama-2-7b-chat-hf
+- meta-llama/Llama-2-13b-hf
+- meta-llama/Llama-2-13b-chat-hf
+- meta-llama/Llama-2-70b-hf
+- meta-llama/Llama-2-70b-chat-hf
+- meta-llama/Meta-Llama-3-8B
+- meta-llama/Meta-Llama-3-8B-Instruct
+- meta-llama/Meta-Llama-3-70B
+- meta-llama/Meta-Llama-3-70B-Instruct
+
+### What if I want to use another dataset? 
+
+The training configs provided in this template all train on the GSM8k which requires a context length of 512 tokens.
+How to ensure the correct format is described in https://docs.endpoints.anyscale.com/fine-tuning/dataset-prep.
+You can replace the s3 buckets in the training configs in this template with paths to your own dataset.
+
+### I have the right model, context length and everything. Can I optimize compute cost?
+
+Optimizing your fine-tuning runs for compute cost is a non-trivial problem.
+The default configs in this template require the following compute:
+Llama-3-8B and Mistral require 16 A10Gs. Llama-3-70B and Mixtral require 32 A10Gs.
+If you want different compute, we *suggest* the following workflow to find a suitable configuration:
+
+* Select some model, context length, fine-tuning technique (LoRA or full-paramter), etc. that suit the problem you want to solve
```

Comment:
emphasize the context length required for the problem setting is important and it impacts how much batch size you can fit. Batchsize is determined for the worst case and you want to make sure at largest batch size you don't oom. Mention that if they have small number of data points that required huge context lenghts, see if you can remove those data points to reduce context length varations.